### PR TITLE
add eslint config to repo to integrate with text editors

### PR DESCRIPTION
**Pull request by JakeChampion**

### CODE REVIEW

**Comment by origamiserviceuser** [comment]
o-expander bundle size difference from 5.0.3 to df088ea6ed8fbaf8ef47483d12014b328564fde2
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-expander bundle size difference from 5.0.3 to ec06ef536e325ab055f65667ee77f7f39602cf44
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-expander bundle size difference from 5.0.3 to afe0ad67d7037ea07e8c67647dce7af381391a8f
No significant bundle size differences found.

**Comment by origamiserviceuser** [comment]
o-expander bundle size difference from 5.0.3 to 5bd7d283c9d213e5480db2218cc5c1e94d22b2ee
No significant bundle size differences found.

File: src/js/expander-utility.js [hunk 1]
```diff
@@ -48,7 +48,7 @@ class ExpanderUtility {
 		// If `shrinkTo` is a number, cast to an actual number using the
 		// unary operator `+`. I.e so `typeof` returns `number`.
 		if (!isNaN(this.options.shrinkTo)) {
-			this.options.shrinkTo = +this.options.shrinkTo;
+			this.options.shrinkTo = Number(this.options.shrinkTo);
```
Comment (notlee) [inline src/js/expander-utility.js]: nice 👍

**Comment by origamiserviceuser** [comment]
o-expander bundle size difference from 5.0.3 to 724cc3e20f825214a9f8c44b1f9f1cd621b0e86e
No significant bundle size differences found.

**Pull request merged**